edx_oauth2_provider/__init__.py

```diff
@@ -1,6 +1,7 @@
 """
 OAuth2 provider based on `django-oauth2-provider` with customizations
 for the edx-platform.
-
 """
+from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
Right - the only thing needed to pass quality here was `unicode_literals` for the version string below. I just copied all these `__future__` imports everywhere as a blunt instrument. Willing to whittle down the list if desired.